_attributes/attributes-openshift-dedicated.adoc

```diff
@@ -13,3 +13,10 @@
 :AWS: Amazon Web Services (AWS)
 :GCP: Google Cloud Platform (GCP)
 :kebab: image:kebab.png[title="Options menu"]
+:rhoda: Red Hat OpenShift Database Access
+:rhods: Red Hat OpenShift Data Science
+:rhoda-short: OpenShift Database Access
+:rhods-short: OpenShift Data Science
+:os: OpenShift
```

Comment:
Sorry - when I said we couldn't use "OpenShift" by itself and should instead use an attribute, I didn't mean to make one for it. I meant that it's normally improper to use the word "OpenShift" by itself, and should instead have a proper product name like "OpenShift Container Platform" or "OpenShift Dedicated" or "OKD", which we happen to have attributes for.

Can you remove this attribute? And then for where you're using it - I tried looking through the ROSA docs to see what you guys are doing, but it seems to vary. I'd recommend using "Red Hat OpenShift" in its place for now.

Reply:
I have adjusted the text as requested.